File: python_POO/CursoPOOUber/Java/Card.java
class Card extends Payment {
    Integer card_number;
    String card_date;
    Integer card_cvv;
    
    public Card (Integer id, Integer amount, Integer card_number, String card_date, Integer card_cvv){
        super(id, amount);
        this.card_number = card_number;
        this.card_date = card_date;
        this.card_cvv = card_cvv;
    }
}

File: python_POO/CursoPOOUber/PHP/Card.php
<?php
require_once("Payment.php");

    class Card extends Payment {
        public $card_number;
        public $card_date;
        public $card_cvv;

        public function __construct($id, $amount, $card_number, $card_date, $card_cvv){
            parent::__construct($id, $amount);
            $this->card_number = $card_number;
            $this->card_date = $card_date;
            $this->card_cvv = $card_cvv;
        }
    }
?>